lib/salus/scanners/osv/base.rb

```diff
@@ -0,0 +1,137 @@
+require 'salus/scanners/base'
+require 'json'
+require 'zip'
+require 'uri'
+require 'net/http'
+
+module Salus::Scanners::OSV
+  class Base < Salus::Scanners::Base
+    DATABASE_STRING_MAPPING = {
+      "GHSA" => "Github Advisory Database",
+      "PYSEC" => "Python Packaging Advisory Database",
+      "GO" => "Go Vulnerability Database",
+      "RUSTSEC" => "RustSec Advisory Database",
+      "default" => "Open Source Vulnerabilitiy"
```

Comment:
Do the OSV ones lack a prefix?  It might be nice to capture unsupported ones too.  We could then setup monitors for "unsupported" entries that would alert us when OSV adds a new datasource and wee need to update the mapping here.  Thoughts?

Reply:
Using default: "Open Source Vulnerability as the "unsupported" catch all. Since OSV pulls from other sources so if that is not mapped I default to "Open Source Vulnerability".